src/main/java/net/logstash/logback/composite/CompositeJsonFormatter.java

```diff
@@ -118,6 +118,56 @@ public boolean isStarted() {
         return started;
     }
 
+    /**
+     * Create a reusable {@link JsonFormatter} bound to the given {@link OutputStream}.
+     * 
+     * @param outputStream the output stream used by the {@link JsonFormatter}
+     * @return {@link JsonFormatter} writing JSON content in the output stream
+     * @throws IOException thrown when unable to write in the output stream or when Jackson fails to produce JSON content
+     */
+    public JsonFormatter createJsonFormatter(OutputStream outputStream) throws IOException {
+        if (!isStarted()) {
+            throw new IllegalStateException("Formatter is not started");
+        }
+        
+        JsonGenerator generator = createGenerator(outputStream);
+        return new JsonFormatter(generator);
+    }
+
+    /**
+     * Create a reusable {@link JsonFormatter} bound to the given {@link Writer}.
+     * 
+     * @param writer the {@link Writer} used by the {@link JsonFormatter}
+     * @return {@link JsonFormatter} writing JSON content in the writer
+     * @throws IOException thrown when unable to write in the writer or when Jackson fails to produce JSON content
+     */
+    public JsonFormatter createJsonFormatter(Writer writer) throws IOException {
```

Comment:
Is this method used anywhere?

Reply:
It was initially used by the CompositeLayout. Not used since I decided to not spend anymore time optimising the Layouts... Can be removed.